coordinator/src/trade/mod.rs

```diff
@@ -672,8 +672,13 @@ impl TradeExecutor {
         trade_params: &TradeParams,
         resize_action: ResizeAction,
     ) -> Result<()> {
-        if !self.node.inner.is_dlc_channel_confirmed(&dlc_channel_id)? {
-            bail!("Underlying DLC channel not yet confirmed");
+        if !self
+            .node
+            .inner
+            .is_dlc_channel_confirmed(&dlc_channel_id)
+            .await?
+        {
+            bail!("Did not see funding tx in mempool yet.");
```

Comment:
🔧 I don't like this change because it assumes that the DLC channel will always be confirmed as soon as the transaction hits mempool, but this may not always be the case.

I think the original error message is fine, but if you want better error messages you could adapt the `is_dlc_channel_confirmed` API to return more information to build a better error message here.

Reply:
> 🔧 I don't like this change because it assumes that the DLC channel will always be confirmed as soon as the transaction hits mempool, but this may not always be the case.

I thought this was the case. When do we have to wait for a confirmation?

Reply:
What I mean is that the error message now reflects an implementation detail: that the number of required confirmations is always 0. And the implementation detail may change in the future, but this error message is independent of that and could easily get stale.

Reply:
Fair enough! I will adapt the message.